src/pages/Account/Venue/VenueMapEditor/DraggableSubvenue.tsx

```diff
@@ -0,0 +1,120 @@
+import React, { CSSProperties, useEffect } from "react";
+import { DragSourceMonitor, useDrag } from "react-dnd";
+import { getEmptyImage } from "react-dnd-html5-backend";
+import { Resizable } from "re-resizable";
+
+import { Dimensions } from "types/utility";
+
+import { SubVenueIcon } from "./Container";
+import { ItemTypes } from "./ItemTypes";
+
+import styles from "./DraggableSubvenue.module.scss";
+
+const getStyles = (
+  left: number,
+  top: number,
+  isDragging: boolean
+): React.CSSProperties => ({
+  position: "absolute",
+  top,
+  left,
+  opacity: isDragging ? 0 : 1,
+  height: isDragging ? 0 : "",
+  background: "rgba(147, 124, 99, 0.2)",
+});
```

Comment:
This function is problematic, not only that there alternatives to it in `useCss()` and similar hooks, but also because its value is used directly for `style={}` which will probably trigger re-render.

That being said, I'm guessing it's not a big issue since dragging is limited to rare occasions of edit in Admin and it may probably provide for a desired side-effect to have components re-render on drag.